homeassistant/components/sonarr/__init__.py

```diff
@@ -1,10 +1,6 @@
 """The Sonarr component."""
-from __future__ import annotations
```

Comment:
I suggest to add it back. It does provide a small performance improvement when type annotations are used. `from __future__ import annotations` could also be added to `sonarr/entity.py`.